src/components/feedback/Dialog.tsx

```diff
@@ -152,31 +190,44 @@ const DialogNext = function Dialog({
   );
 
   return (
-    <div
-      data-component="Dialog"
-      tabIndex={-1}
-      // NB: Role can be overridden with an HTML attribute; this is purposeful
-      role="dialog"
-      {...htmlAttributes}
-      className={classnames(
-        // Column-flex layout to constrain content to max-height
-        'flex flex-col',
-        classes
-      )}
-      ref={downcastRef(modalRef)}
+    <Wrapper
+      visible={visible}
+      onTransitionEnd={() => {
```

Comment:
Ah, interesting, so we can rely on an `onTransitionEnd` event for elements with no transition defined? I'll have to look that up!